src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margin/paddings for all elements, only for elements that you actually need